styles.js

```diff
@@ -39,9 +39,17 @@ export const StyledList = styled.ul`
 export const StyledItem = styled.li`
   border: 1px solid;
   margin: 10px 5px 10px 5px; //im uhrzeigersinn beginnend top
```

Comment:
```suggestion
  margin: 10px 5px 10px 5px;
```
Please don't leave comments in the code you want to deploy. 🙏